app/models/test.rb

```diff
@@ -1,8 +1,13 @@
 # frozen_string_literal: true
 
 class Test < ApplicationRecord
+  belongs_to :category
+  belongs_to :author, class_name: "User", foreign_key: 'user_id'
+  has_many :questions, dependent: :destroy
+  has_many :test_stats, dependent: :destroy
+
   def self.show_by_category_name(value)
-    Test.order(title: :desc).joins('LEFT JOIN categories ON tests.category_id = categories.id')
+    Test.order(title: :desc).joins(:category)
         .where('categories.title = :value', value: value).pluck(:title)
```

Comment:
```suggestion
        .where(categories: { title: value }).pluck(:title)
```
"хеш-синтаксис" лаконичней и его обычно легче редактировать

Reply:
Рубокоп